libs/ui/packages/react/src/components/asorted/Icon/ProviderIcon.tsx

```diff
@@ -15,8 +17,24 @@ export type ProviderSizes = keyof typeof sizes;
 export type Props = {
   name: string;
   size?: ProviderSizes;
+  boxed?: boolean;
 };
 
+// @ts-expect-error r is webpackContext
+function importAll(r) {
+  return r.keys().map(r);
+}
+
+const getFilenameFromPath = (path: string) => path.replace(/^.*[\\/]/, "").slice(0, -4);
+
+const providersFavicons = importAll(
+  // @ts-expect-error require not found
+  require.context("./ProvidersFavicons", false, /\.(png|jpe?g|svg)$/),
```

Comment:
Sorry but it is a very bad idea to incorporate webpack specific code in a library.
I know it will complicate things but you need to find another way of importing fav icons.